### PR TITLE
bulk-cdk-toolkits-extract-cdc: add CustomConverter utils

### PR DESCRIPTION
**Pull request by postamar**
## What
This PR introduces new CustomConverter utilities in the Bulk CDK to cut down on the boilerplate.
See https://github.com/airbytehq/airbyte/pull/49918 for a use case

## User Impact
None

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/PartialConverter.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+package io.airbyte.cdk.read.cdc
+
+import io.debezium.spi.converter.ConvertedField
+import io.debezium.spi.converter.CustomConverter
+import io.debezium.spi.converter.RelationalColumn
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.util.concurrent.atomic.AtomicBoolean
+
+fun interface PartialConverter {
```
Comment (theyueli) [inline airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/PartialConverter.kt:13]: what is a partial converter? could we have some documentation?

Comment (postamar) [reply airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/PartialConverter.kt:13]: this definitely needs comments, sorry about that

Comment (postamar) [reply airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/PartialConverter.kt:23]: done

**Review by xiaohansong — APPROVED** [review]
Should we also use this for mysql custom converter, just to test and set an example?

**Pull request merged**